Signal/src/ViewControllers/AppSettingsViewController.m

```diff
@@ -168,6 +170,13 @@ - (void)updateTableContents
         }
                                                    actionBlock:nil]];
     }
+
+    [section addItem:[OWSTableItem
+                         disclosureItemWithText:NSLocalizedString(@"PROFILE_VIEW_TITLE", @"Title for the profile view.")
```

Comment:
That's a really great idea.  I made a card:

https://trello.com/c/hOTJGrVN/1268-restyle-app-settings-views-header-to-reflect-users-profile